mautic/mautic
# TPROD-297 Use __serialize() and __unserialize() instead  serialize() and unserialize()

…alize()



| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [n]
| New feature/enhancement? (use the a.x branch)      | [n]
| Deprecations?                          | [y]
| BC breaks? (use the c.x branch)        | [n]
| Automated tests included?              | [n] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | https://mautic.atlassian.net/browse/TPROD-297



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Verify saving plugin credentials and checking those are working fine.




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11150"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
